README.md

```diff
@@ -81,12 +81,12 @@ You need [CMake](https://cmake.org/download/) to configure the Exiv2 project, an
 ## Build, Install, Use Exiv2 on a UNIX-like system
 
 ```bash
-$ cd ~/gnu/github/exiv2                     # Location of the project code
-$ mkdir build && cd build                   # Create a build directory
-$ cmake -DCMAKE_BUILD_TYPE=Release ..       # Configure the project with CMake
-$ cmake --build .                           # Compile the project
-$ ctest --verbose                           # Run tests
-$ cmake --install .                         # Run the install target (install library, public headers, application and CMake files)
+$ cd ~/gnu/github/exiv2                          # Location of the project code
+$ rm -r build                                    # Remove existing build directory
```

Comment:
rm -r does not remove directories

Reply:
There is no spot on the README that uses `-f`. What benefit does it offer developers?

Reply:
There was `rm -rf build` just above that you removed, no reason to change it.

Reply:
I'll remove this line. I really don't think it's needed, it's already written above. It's fixed in 225cb92a6ebe5ba72e29d6e582a8f44b5340e2f0